test/cypress/integration/dialog-extension.spec.js

```diff
@@ -76,6 +80,13 @@ context('Dialog extension', () => {
     })
   })
 
+  it('sdk.parameters have expected values', () => {
+    cy.getSdk(iframeDialogSelector).then(sdk => {
```

Comment:
Are these tests able to handle async code without any intervention? If not, these tests aren't actually running.
What I mean by this is the test would need to look something like this:

```js
it('sdk.parameters have expected values', async () => {
await   verifySdkInstallationParameters(iframeDialogSelector)
await   verifySdkInvocationParameters(iframeDialogSelector)
})
```

or

```js
it('sdk.parameters have expected values', (done) => {   
verifySdkInstallationParameters(iframeDialogSelector).then(done)
verifySdkInvocationParameters(iframeDialogSelector).then(done)
})
```

Reply:
@davidfateh https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Chains-of-Commands

> If you’re a modern JS programmer you might hear “asynchronous” and think: why can’t I just use async/await instead of learning some proprietary API? Cypress’s APIs are built very differently from what you’re likely used to.

Reply:
Thanks Alex, didn't know that!